app/src/components/common/wrong_network_message/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { RouteComponentProps } from 'react-router'
+import { withRouter } from 'react-router-dom'
+
+import { SectionTitle } from '../section_title'
+import { ViewCard } from '../view_card'
+import { SubsectionTitle } from '../subsection_title'
+import { ButtonContainer } from '../button_container'
+import { ButtonLink } from '../button_link'
+import { networkIds } from '../../../util/networks'
+
+const WrappedText = styled.div`
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.36;
+  letter-spacing: normal;
+  text-align: left;
+  color: #555555;
+  margin-top: 20px;
+  margin-bottom: 10px;
+`
+
+const ButtonLinkStyled = styled(ButtonLink)`
+  margin-right: auto;
+`
+
+interface Props extends RouteComponentProps<{}> {
+  theme?: any
+}
+
+const WrongNetworkMessageContainer = (props: Props) => {
+  return (
+    <>
+      <SectionTitle title="THERE WAS A PROBLEM..." />
+      <ViewCard>
+        <SubsectionTitle>Wrong network configuration</SubsectionTitle>
+        <WrappedText>
+          We couldn&apos;t find the network with the provided wallet configuration.
```

Comment:
I'm pretty sure (but not 100% sure) that you can use `'` in JSX.

Reply:
We will incur in this linter error 
![Screenshot_20200204_183837](https://user-images.githubusercontent.com/1144028/73789529-b9170880-477d-11ea-80c7-df36a33eed0b.png)